src/App.tsx

```diff
@@ -1,61 +1,117 @@
+import React, { useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { Todo } from './Types/Todo';
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(todosFromServer);
+
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState('');
+
+  const [isTitleError, setIsTitleError] = useState(false);
+  const [isUserError, setIsUserError] = useState(false);
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+
+    setIsTitleError(false);
+  };
+
+  const handleUserChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(event.target.value);
+
+    setIsUserError(false);
+  };
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+  };
+
+  const getUsersMaxId = (TodoItems: Todo[]) => {
+    return Math.max(...TodoItems.map((todoItem) => todoItem.id));
+  };
+
+  const getNewTodo = (currentTodoList: Todo[]) => ({
+    id: getUsersMaxId(currentTodoList) + 1,
+    title,
+    completed: false,
+    userId: +userId,
+  });
+
+  const setNewTodo = () => {
+    setTodos((currentTodos) => [...currentTodos, getNewTodo(currentTodos)]);
+
+    setTitle('');
+    setUserId('');
+  };
+
+  const addTodo = () => {
+    if (userId && title) {
```

Comment:
spaces as a todo's name are still acceptable 😢 
let's `trim` the `title` and check it's length 😉 
<img width="166" alt="image" src="https://user-images.githubusercontent.com/55802630/184633066-80f3e157-2028-4b19-868b-e5eaed773a44.png">